lighthouse-core/lib/asset-saver.js

```diff
@@ -162,9 +162,13 @@ async function prepareAssets(artifacts, audits) {
   for (const passName of passNames) {
     const trace = artifacts.traces[passName];
     const devtoolsLog = artifacts.devtoolsLogs[passName];
+
+    // Avoid Runner->AssetSaver->Runner circular require by loading Runner here.
```

Comment:
:(

Reply:
yeah, really disappointed to get there and run into that. Hopefully with `computedArtifacts` refactor that'll get cleaned up